BAZProjectC1/Models/Strings.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Strings.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 15/09/22.
+//
+
+import Foundation
+
+enum EndpointsList:String  {
```

Comment:
Deja un espacio entre los `:` y el tipo String y solo un espacio entre las llaves
```suggestion
enum EndpointsList: String {
```